OPENSCAP-4930, OPENSCAP-4931, OPENSCAP-4932, OPENSCAP-4933, OPENSCAP-4934, OPENSCAP-4935, OPENSCAP-4936, OPENSCAP-4937, OPENSCAP-4938, OPENSCAP-4939, OPENSCAP-4940, OPENSCAP-4941, OPENSCAP-4942, OPENSCAP-4943, OPENSCAP-4944, OPENSCAP-4945, OPENSCAP-4946, OPENSCAP-4947: Add arch filter to ARPC

jan-cerny opened the pull request:
#### Description:

We should have the `-F arch=` filter in rules that use the `audit_rules_privileged_commands` template. If the system is 64-bit, there should be 2 rules, one with  `-F arch=b32` and one with  `-F arch=b64`.

In this PR we update the  `audit_rules_privileged_commands`  template to have the `-F arch=` filter there. This consist of updating OVAL, remediations and also test scenarios.

Also, we will unify the descriptions of all rules that use this template. This reduces code duplication.

#### Rationale:

Update for RHEL 10.

#### Review Hints:

Use Automatus tests.

Mab879 commented on shared/templates/audit_rules_privileged_commands/kubernetes.template:
```diff
@@ -13,7 +13,7 @@ spec:
       files:
       - contents:
-          source: data:,-a%20always%2Cexit%20-F%20path%3D{{{ PATH }}}%20-F%20auid%3E%3D{{{ auid }}}%20-F%20auid%21%3Dunset%20-F%20key%3Dprivileged%0A
+          source: data:,-a%20always%2Cexit%20-F%20arch%3Db32%20-F%20path%3D{{{ PATH }}}%20-F%20auid%3E%3D{{{ auid }}}%20-F%20auid%21%3Dunset%20-F%20key%3Dprivileged%0A-a%20always%2Cexit%20-F%20arch%3Db64%20-F%20path%3D{{{ PATH }}}%20-F%20auid%3E%3D{{{ auid }}}%20-F%20auid%21%3Dunset%20-F%20key%3Dprivileged%0A
```
This doesn't seem right. 

The part that concerns me the most`-F arch`

jan-cerny replied:
I think you mean the missing perm_x there. I have add it now. But, it won't be used because the rhcos4 product doesn't set this perm_x.

jan-cerny commented:
I have rebased this PR on the top of the latest upstream master branch.

Mab879 commented:
/packit retest-failed

jan-cerny commented:
I have removed the k8s remediation change from this PR. I have extracted the change to a separate PR https://github.com/ComplianceAsCode/content/pull/13258.
